data/payroll.mk

```diff
@@ -0,0 +1,69 @@
+.INTERMEDIATE : %-with-agencies.csv %-with-data-year.csv \
```

Comment:
Almost all of this is copied from `data/amended_files.mk`. The new things are the `import/` and `amend/` targets, and the gut checks in the `with-agencies` and `no-salary-omitted` targets.